proxysql/install-deps

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
@rameshvs02  maybe we can use 'perconalab/pxc-build'  docker image to build proxysql... Could you please check the https://github.com/Percona-Lab/jenkins-pipelines/blob/master/pxc/docker/install-deps maybe we can add a few additional rpms which will not conflict with pxc builds

Reply:
@rameshvs02  ^^^